src/w3fa03v.f

```diff
@@ -1,47 +1,34 @@
 C> @file
-C                .      .    .                                       .
-C> SUBPROGRAM:    W3FA03V     COMPUTE STANDARD HEIGHT, TEMP, AND POT TEMP
-C>   PRGMMR: KEYSER           ORG: W/NMC22    DATE: 92-06-29
+C> @brief Compute standard height, temp, and pot temp.
+C> @author James McDonell @date 1974-06-01
 C>
-C> ABSTRACT: COMPUTES THE STANDARD HEIGHT, TEMPERATURE, AND POTENTIAL
-C>   TEMPERATURE GIVEN THE PRESSURE IN MILLIBARS ( > 8.68 MB ).  FOR
-C>   HEIGHT AND TEMPERATURE THE RESULTS DUPLICATE THE VALUES IN THE
-C>   U.S. STANDARD ATMOSPHERE (L962), WHICH IS THE ICAO STANDARD
-C>   ATMOSPHERE TO 54.7487 MB (20 KM) AND THE PROPOSED EXTENSION TO
-C>   8.68 MB (32 KM).  FOR POTENTIAL TEMPERATURE A VALUE OF  2/7  IS
-C>   USED FOR  RD/CP.
+C> Computes the standard height, temperature, and potential
+C> temperature given the pressure in millibars ( > 8.68 mb ). For
```

Comment:
Consider remove the extra space:
temperature given the pressure in millibars (> 8.68 mb). For

Reply:
Done